include/overlay006/encounter_data.h

```diff
@@ -0,0 +1,33 @@
+#ifndef POKEPLATINUM_WILD_ENCOUNTER_DATA_H
+#define POKEPLATINUM_WILD_ENCOUNTER_DATA_H
+
+#include "overlay006/encounter_field_data.h"
+#include "overlay006/encounter_water_data.h"
+
+typedef struct WildEncounterData {
+    int encounterRateField;
+    FieldEncounterData fieldEncounters[12];
+    int swarmEncounters[2];
+    int dayEncounters[2];
+    int nightEncounters[2];
+    int radarEncounters[4];
+    int encounterRatesForms[5];
+    int unownTableID;
+    int dualSlotRubyEncounters[2];
+    int dualSlotSapphireEncounters[2];
+    int dualSlotEmeraldEncouters[2];
+    int dualSlotFireredEncounters[2];
+    int dualSlotLeafgreenEncounters[2];
+    int encounterRateSurf;
+    WaterEncounterData surfEncounters[5];
+    int unk_CC;
+    WaterEncounterData unk_D0[5];
```

Comment:
Given the placement of these in the data structure relative to Heart Gold, I would posit that these were originally planned to be used for Rock Smash encounters. It might be pertinent to label them as `unused` rather than `unk`.